vagrant/shell/pluton.sh

```diff
@@ -7,12 +7,12 @@ echo "deb https://apt.dockerproject.org/repo ubuntu-xenial main" | sudo tee /etc
 
 # install packages
 sudo apt-get update -y
-sudo apt-get -y install docker-engine python-virtualenv python-pip
+sudo apt-get -y install docker-engine python-virtualenv python3-pip python3.5
```

Comment:
you can use ``python3.5 -m ensurepip`` instead of ``apt-get install python3-pip``

Reply:
this will require installation of ```python3-venv``` package. Do you still thinking this is good change?